api/v1alpha1/flow_webhook.go

```diff
@@ -47,6 +47,21 @@ var _ webhook.Defaulter = &Flow{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *Flow) Default() {
+	if r.Spec.DestinationPorts == nil {
```

Comment:
The user might think, if it is empty, it would do the same thing as: `source-subnets: []`